Hotfix

Edwardb11 opened the pull request.

Edwardb11 committed:
bug bg

Edwardb11 committed:
update

Edwardb11 committed:
flex wrao

Edwardb11 committed:
update flex

Edwardb11 committed:
delete style

Lenugo commented:
Noto que en este #12 solucionaste lo del background, pero eliminaste los estilos en móvil lo cual deberas reestablecer.

### Sobre el desbordamiento de los números:
Tu Problema es que cuando excede la longitud de 14 números los mismos se desbordan, si te fijas en la calculadora de tu celular o de cualquier dispositivo cuando la longitud del resultado excede los 9 números se agrega una letra `e` la cual significa exponente, acá una [lectura](https://www.geniolandia.com/13168095/que-significa-la-e-al-final-de-un-numero) sobre el tema. Puedes agregar esa notación con _javascript_ a tu código, y asegurarte que se vea bien en los dos dispositivos en los cuales estas trabajando.
![tip-calculator-app1-rni4r41po-edwardb11 vercel app_](https://user-images.githubusercontent.com/54646168/152182314-4c0e4610-7e7c-4d65-85c2-535841a3cae3.png)



### Otros tips:
> - Aumentale el *font-size* al placeholder para que ocupe el espacio completo en la vista desktop.
> ![FireShot Capture 241 - Tip calculator app - tip-calculator-app1-rni4r41po-edwardb11 vercel app](https://user-images.githubusercontent.com/54646168/152182577-29a5cf91-80b2-4328-83c5-88d56bb291ca.png)
> - Para que el **main** se note mas centrado en la versión desktop, puedes intentar darle un mayor *mayor-bottom* al header.
> ![FireShot Capture 242 - Tip calculator app - tip-calculator-app1-i6ybd73q9-edwardb11 vercel app](https://user-images.githubusercontent.com/54646168/152183183-cc38c692-ea3f-4cc6-9c21-c36871c31ad6.png)

Edwardb11 committed:
update responsive

Edwardb11 committed:
center placeholder

Edwardb11 committed:
add margin buttom to main

Lenugo commented:
> Estuve revisando los recientes commits, acá un [vídeo](https://www.loom.com/share/860a9c7cc3dd4b15b85d993e38bab2f3) con sugerencias.

Edwardb11 committed:
notacion cientifica

Edwardb11 committed:
add margin top to main select

Lenugo commented:
Vamos por buen camino, solo quedarian de mi parte dos fix a considerar: 

1. Dismuniur el ancho del *main* y por consiguiente quitar el padding left del `custom field` (screen tomado con la resolución de `1130 x 881`. 
> ![tip-calculator-app1-z9zlyukk3-edwardb11 vercel app_ (1)](https://user-images.githubusercontent.com/54646168/152538137-5e6789c2-5fbc-4865-8395-ba689ee9175f.png)

2. En la transición entre  `1178 y 1024` la aplicación se desborda  
 > ![tip-calculator-app1-z9zlyukk3-edwardb11 vercel app_ (3)](https://user-images.githubusercontent.com/54646168/152539188-6eaea901-992d-4b7d-8cb8-ed685b635b1c.png)

Edwardb11 committed:
update the left margin of the custom button and round the notation to 2

Lenugo commented on src/componets/box/Box.js:
```diff
@@ -29,13 +29,13 @@ export default function Box() {
         setTotal(total);
         setTipC("");
       }
-      if (total.length >= 9 || TipAmount.length >= 9) {
```
No es necesario que disminuyas el tipAmount, la notacion debe presentarse cuando la longitud excedera 9 digitos

Edwardb11 replied:
Done

Edwardb11 committed:
update conditional and add mx-1 and mx-auto in flex

Edwardb11 committed:
Eliminating the To Exponential
Adding limit to input

Changes to be committed:
	modified:   src/componets/box/Box.js

Author
    @Edwardb11

Edwardb11 committed:
add scientific notation again because I realized the previous mistake was mine and modify/remove some properties

Edwardb11 committed:
updating the main suggestion about the size and only make the responsive in md and sm

Lenugo commented on src/componets/box/Box.js:
```diff
@@ -52,17 +41,6 @@ export default function Box() {
-        setTipAmount(TipAmount);
-        setTotal(total);
-        setTip("");
-      }
```
Convencionalmente en las calculadoras la letra 'e' se agrega desde el décimo número en adelante. lo cual no esta configurado así. Deberías volver a dejar la configuración que tenia antes pero, con los nuevos cambios en la UI.
![FireShot Capture 247 - Tip calculator app - tip-calculator-app1-i04sakt9u-edwardb11 vercel app](https://user-images.githubusercontent.com/54646168/153457452-3be57a3f-95ba-4e73-b703-6a992b5539a3.png)

Edwardb11 replied:
Done

Lenugo reviewed (commented):
Nice

Edwardb11 committed:
scientific notation from 10 digits onwards

Edwardb11 committed:
max 100

Lenugo reviewed (approved):
LGTM
